.env-dist

```diff
@@ -31,4 +31,5 @@ PREMIUM_PROD_ID=prod_K29ULZL9pUR9Fr
 SUBSCRIPTIONS_WITH_UNLIMITED="monitor-unlimited,mozilla-one,guardian_vpn"
 PREMIUM_RELEASE_DATE='2021-10-22 17:00:00+00:00'
 MAX_ONBOARDING_AVAILABLE=3
-DOMAIN_REGISTRATION_MODAL=True
\ No newline at end of file
+DOMAIN_REGISTRATION_MODAL=True
+SERVE_REACT=False
```

Comment:
Again, set this to `True` to actually serve the app in the frontend. We can also use this when we're ready to go live.